### PR TITLE
[RIP-9] Add English document for Design_LoadBlancing.md and Design_Remoting.md

### PR DESCRIPTION
**Pull request by universefeeler**
## What is the purpose of the change

add Design:Load blancing for docs/en

## Brief changelog

add en load blancing doc

## Verifying this change

universefeeler

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/en/7.Design:Load_blancing.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## Load Balancing
+Load balancing in RocketMQ is accomplished on Client side. Specifically, it can be divided into load balancing at Producer side when sending messages and load balancing at Constumer side when subscribing messages.
+
+### Producer Load Balancing
```
Comment (wlliqipeng) [inline docs/en/7.Design:Load_blancing.md]: 1 Producer Load Balancing

File: docs/en/7.Design:Load_blancing.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+When the Producer sends a message, it will first find the specified TopicPublishInfo according to Topic. After getting the routing information of TopicPublishInfo, the RocketMQ client will select a queue (MessageQueue) from the messageQueue List in TopicPublishInfo  to send the message by default.Specific fault-tolerant strategies are defined in the MQFaultStrategy class.
+Here is a sendLatencyFaultEnable switch variable, which, if turned on, filters out the Broker agent of not available on the basis of randomly gradually increasing modular arithmetic selection. The so-called "latencyFault Tolerance" refers to a certain period of time to avoid previous failures. For example, if the latency of the last request exceeds 550 Lms, it will evade 3000 Lms; if it exceeds 1000L, it will evade 60000 L; if it is closed, it will choose a queue (MessageQueue) to send messages by randomly gradually increasing modular arithmetic, and the latencyFault Tolerance mechanism is the key to achieve high availability of message sending.
+
+### Consumer Load Balancing
```
Comment (wlliqipeng) [inline docs/en/7.Design:Load_blancing.md]: 2 Consumer Load Balancing

File: docs/en/7.Design:Load_blancing.md (hunk 1 above)
Comment (universefeeler) [reply docs/en/7.Design:Load_blancing.md]: ok, added it

File: docs/en/7.Design:Load_blancing.md (hunk 2 above)
Comment (universefeeler) [reply docs/en/7.Design:Load_blancing.md]: ok, added it, please review again

**Pull request merged**